pkg/scheduler/scheduler.go

```diff
@@ -369,14 +368,14 @@ func (s *Scheduler) doScheduleBinding(namespace, name string) (err error) {
 	if appliedPlacement := util.GetLabelValue(rb.Annotations, util.PolicyPlacementAnnotation); policyPlacementStr != appliedPlacement {
 		// policy placement changed, need schedule
 		klog.Infof("Start to schedule ResourceBinding(%s/%s) as placement changed", namespace, name)
-		err = s.scheduleResourceBinding(rb)
+		err = s.scheduleResourceBinding(rb, policyPlacement, policyPlacementStr)
```

Comment:
Is this change mandatory for the cherry-pick? Can we have a minimum patch?

Reply:
> Is this change mandatory for the cherry-pick? Can we have a minimum patch?

it is related, I don't want to call `DeepCopy` in multiple places to copy Placement.

Reply:
The first thing we need to figure out is if the same bug exists in the release branch, and we should always do minimum updates against the release branch to avoid introducing additional bugs.